pkg/channel/consolidated/reconciler/controller/controller.go

```diff
@@ -174,22 +159,23 @@ func NewController(
 			knativeReconciler.LabelFilterFunc(roleLabelKey, dispatcherRoleLabelValue, false),
 		),
 		Handler: cache.ResourceEventHandlerFuncs{
-			// Cancel probing when a Pod is deleted
-			DeleteFunc: getPodInformerEventHandler(ctx, logger, statusProber, impl, kafkaChannelInformer, "Delete"),
-			AddFunc:    getPodInformerEventHandler(ctx, logger, statusProber, impl, kafkaChannelInformer, "Add"),
+			// TODO: do we still need this?
+			// Global sync when a Pod is deleted or added
+			DeleteFunc: getPodInformerEventHandler(ctx, logger, impl, kafkaChannelInformer, "Delete"),
+			AddFunc:    getPodInformerEventHandler(ctx, logger, impl, kafkaChannelInformer, "Add"),
```

Comment:
It was introduced as a change on top of adding the prober... which we now remove, hence I _think_ we can remove the hndler here

Reply:
Done